Content.Shared/Singularity/Components/ContainmentFieldGeneratorComponent.cs

```diff
@@ -59,7 +59,7 @@ public int PowerBuffer
     /// How many tiles should this field check before giving up?
     /// </summary>
     [DataField("maxLength")]
-    public float MaxLength = 8F;
+    public float MaxLength = 16F;
```

Comment:
Dont do this in the same PR
Also need to add a note about it being frontier change

# Frontier

Reply:
what about a frontier change? Also, noted, I will make another PR for containment field generators